### PR TITLE
adding_a_cask.md: updates for deprecated commands

### PR DESCRIPTION
**Pull request by miccal**
Various updates for deprecated `brew cask` commands.

### CODE REVIEW

File: doc/development/adding_a_cask.md [hunk 1]
```diff
@@ -170,27 +170,27 @@ Give it a shot with:
...
 ```bash
-brew cask uninstall my-new-cask
+brew uninstall my-new-cask
 ```
 
 If everything looks good, you’ll also want to make sure your Cask passes audit with:
 
 ```bash
-brew cask audit my-new-cask --download
+brew audit my-new-cask --download
```
Comment (reitermarkus) [inline doc/development/adding_a_cask.md:187]: `--download` doesn't exist for `brew audit`.

Comment (reitermarkus) [inline doc/development/adding_a_cask.md:187]:
```suggestion
brew audit my-new-cask --new-cask
```

Comment (miccal) [reply doc/development/adding_a_cask.md]: Is `brew audit --new-cask my-new-cask` ok?

Comment (reitermarkus) [reply doc/development/adding_a_cask.md]: Yes. Doesn't matter where the flag is.

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**